opal/mca/btl/ofi/btl_ofi_component.c

```diff
@@ -213,6 +226,20 @@ void mca_btl_ofi_exit(void)
     exit(1);
 }
 
+static int is_in_list(char * const *list, const char *item)
```

Comment:
Nit: if the function is actually returning a boolean value, please make the return type be `bool` / `true` or `false`.